packages/web-components/package.json

```diff
@@ -24,6 +24,14 @@
       "types": "./dist/dts/index.d.ts",
       "default": "./dist/esm/index.js"
     },
+    "./accordion/accordion": {
+      "types": "./dist/esm/accordion/accordion/define.d.ts",
+      "default": "./dist/esm/accordion/accordion/define.js"
+    },
+    "./accordion/accordion-item": {
+      "types": "./dist/esm/accordion/accordion-item/define.d.ts",
+      "default": "./dist/esm/accordion/accordion-item/define.js"
+    },
```

Comment:
Note here as well that I'd prefer we maintain a flat structure - `[component]/[filename].ts`